scalyr_agent/log_processing.py

```diff
@@ -1817,6 +1824,10 @@ class LogLineRedacter(object):
     Redaction rules can match each line multiple times.
     """
 
+    # Indicator in the replacement text[optional] to hash the group content.
+    # eg replacement string of foo\\1=$HG2 will replace the second group with its hashed content
+    HASH_GROUP_INDICATOR = "$HG"
```

Comment:
To be consistent with the normal replacement syntax, how about we call this `\\H`?

So, you would say something like `foo\\1=\\H2`?